docs/_samples/gssapi.py

```diff
@@ -0,0 +1,32 @@
+from pylibsshext.errors import LibsshSessionException
+from pylibsshext.session import Session
+
+
+ssh = Session()
+
+HOST = 'CHANGEME'
+USER = 'CHANGEME'
+TIMEOUT = 30
+PORT = 22
+try:
+    ssh.connect(
+        host=HOST,
+        user=USER,
+        timeout=TIMEOUT,
+        port=PORT,
+        # These parameters are not necessary, but can narrow down which token
+        # should be used to connect, similar to specifying a ssh private key
+        # gssapi_client_identity="client_principal_name",
+        # gssapi_server_identity="server_principal_hostname",
+    )
+except LibsshSessionException as ssh_exc:
+    print(f'Failed to connect over SSH: {ssh_exc!s}')
+
+print(f'{ssh.is_connected=}')
+if ssh.is_connected:
+    chan_shell = ssh.invoke_shell()
+    data_b = chan_shell.read_bulk_response(timeout=2, retry=10)
+    chan_shell.close()
```

Comment:
I think that when showcasing any code with closable resources, it's best to advertise practices of calling `.close()` in a robust manner like
```suggestion
    try:
        data_b = chan_shell.read_bulk_response(timeout=2, retry=10)
    finally:
        chan_shell.close()
```
@Qalthos WDYT?